libs/execution_context/ebpf_program.c

```diff
@@ -215,7 +215,7 @@ _ebpf_program_program_info_provider_changed(
  * @param[in] object Pointer to ebpf_core_object_t whose ref-count reached zero.
  */
 static void
-_ebpf_program_free(ebpf_core_object_t* object)
+_ebpf_program_free(_In_opt_ _Post_invalid_ ebpf_core_object_t* object)
```

Comment:
nit: should we use this or `_Frees_ptr_opt_`. Asking as at some places, `_In_opt_ _Post_invalid_` is used, and on other places, `_Frees_ptr_opt_`

Reply:
I tried this earlier and got a compiler or analysis error so changed it back to this to pass